### ISSUE TITLE
Change the name?

### ISSUE THREAD

**Issue by Plasmmer** [kripken/BananaBread#65]
I think another name can be better than "BananaBread".

**Comment by Croydon** [comment]
For what purposes?

**Comment by Plasmmer** [comment]
Have a exclusive name.

**Comment by kriekprinsloo** [comment]
I agree.  The engine looks similar to the Quake engine in my opinion.

Here's a list of enemies from Quake, Quake 2, and Quake 4:

http://quake.wikia.com/wiki/Monsters_(Q1)

<https://mail.google.com/mail/u/0/goog_1683510813>
http://quake.wikia.com/wiki/Monsters_(Q2)

http://quake.wikia.com/wiki/Category:Quake_IV_enemies

Personally, I like the word Strogg, but that's just me.

**Comment by Plasmmer** [comment]
I agree with a more exclusive name. And a name that you can search on Google, and see the game in the first page. But the actual name....
<img src="http://image.prntscr.com/image/41cc166a7cb849399306aace1aa97bf3.png"/>

**Comment by Croydon** [comment]
The name is out there for several years. I can't see a real benefit from that, but rather too much broken references and work for nothing.

Everyone will find this project if searched together with a keyword like "game", "engine", "browser", "mozilla", "emscripten" or something similar.

**Comment by Plasmmer** [comment]
I'm not talking here about what I want, but about what is better.

**Comment by Croydon** [comment]
Also it's the Cube 2: Sauerbraten engine, naming it after something from Quake would be really discrediting towards the Cube 2 developers.

**Comment by Plasmmer** [comment]
BananaBread wasn't the original name (Cube 2), but the port name. This can be renamed. But use a name from Quake, I don't agree.

**Comment by chasester** [comment]
@name
I like the name, I dont know if I agree with your claim, if you type in cube you dont get the cube engine 👍I dont think this game has much repor so the name isnt gonnae do anythingg but confuse people.

**Comment by Plasmmer** [comment]
Why keep this only a demo with a random name, if this can be a great game?
Because the developers of paid games don't want it?!

**Comment by kriekprinsloo** [comment]
Wait... Why was the name BananaBread chosen in the first place?

**Comment by Plasmmer** [comment]
Its just a random name for a random game. I don't believe this can be a random game, but a great game, like you said.

**Comment by kriekprinsloo** [comment]
Sure, of course it can be a random game with a random name **just because**, but it can also be a good engine, with a great name, that's actually memorable, not named after one of my favorite things in life.

**Comment by Plasmmer** [comment]
But I believe some owners of paid games don't want open-source games being great ;)
Like I don't believe the creators of Photoshop likes GIMP, or creators of ACID likes Audacity, or creators of AutoCAD likes Blender.

**Comment by kriekprinsloo** [comment]
And why wouldn't they?  They already have access to the closed-source applications.  But for the rest of us...  We need to make the **good** open-source projects **great**, or we'll be left in the dust, so to speak.

**Comment by Plasmmer** [comment]
But this is the intention of a lot of owners of paid products. Because it we don't have yet a democratic git service, or a lot of investment on open-source projects.

**Comment by kriekprinsloo** [comment]
Sometimes I like to think that the open-source community is bigger and better than one of the big companies like Adobe...

Anyway, here's a start:

[http://www.fantasynamegenerators.com/game-engine-names.php](http://www.fantasynamegenerators.com/game-engine-names.php )

**Comment by Plasmmer** [comment]
Is bigger and better than a lot of companies/projects/products, but isn't too united.

**Comment by kriekprinsloo** [comment]
Heh. Kind of like me...  But not this time. 

Here's some names I found:

Particle Engine
Havoc Toolset
Fusion Studio
Hysteria Engine
Hex engine

I will keep a running list in this post.  Feel free to mix and match the names!

**Comment by Plasmmer** [comment]
I don't like the Engine name in a game. Engine is a backend. A name need to figure the frontend.

**Comment by kriekprinsloo** [comment]
Hmm... What about SimplyFatal as a name?

**Comment by Plasmmer** [comment]
Please search on Google. Already exists a game with such name.

**Comment by kripken** [comment]
> Wait... Why was the name BananaBread chosen in the first place?

Sauerbraten is a German food, so we were looking for a food name ;) BananaBread seemed as good as any.

I think it has reasonable discoverability, searching for "BananaBread WebGL" gives this repo as the first link.

But if there is going to be new development here, and there is a good name proposal, that sounds great of course. The name new can be like the "2.0" of this project.

**Comment by kriekprinsloo** [comment]
I did.  QuickFire's taken, but I didn't find that SimplyFatal was taken.

**Comment by chasester** [comment]
Whats in a name? A rose by any other name would smell just a sweet.

**Comment by kriekprinsloo** [comment]
If you insist on keeping the name a food, [here's](https://en.wikipedia.org/wiki/List_of_German_dishes) a reliable source for German dishes

**Comment by stkrzysiak** [comment]
May I suggest alternative 'quick' breads: http://www.food.com/topic/quick-breads 
The real question is which bread is the most exclusive? Which bread conveys reliability, performance, and a strong community? I vote for 'BeerBread'

**Comment by profkittyface** [comment]
PlasmaBananaBread

**Comment by profkittyface** [comment]
Breadngine

**Comment by profkittyface** [comment]
Visual Banana Bread Studio 2020

**Comment by profkittyface** [comment]
QuickBread, LoafMaker, VisualLoaf, Glutenizer, Stone Hearth, Hearthstone, BakeThisAt350, SoMuchButter, UnrealLoaf, TemporoBanana, AreTheBlackBitsSpiderEggs

**Comment by ruzz311** [comment]
Can it get a GF badge so my wife with celiac can use it?

**Comment by oxr463** [comment]
I ran into this project quite a few years ago, and I was telling a colleague about it the other day, and the only reason I could remember the name was because of how quirky it was.

+1 for BananaBread!
